server/src/index.ts

```diff
@@ -22,6 +26,10 @@ app.use((req, res, next) => {
 app.use(expressLogger);
 app.use("/", baseRouter);
 
+cron.schedule(crontab, () => {
+    scraper();
```

Comment:
I guess we can add the db related stuff here @resurreccionl , since the scraper has to finish first

Reply:
i don't have access to my computer right now, but yup, i was thinking that too! `setupDb` needs to be modified a bit though, because i think it doesn't insert any values if the tables already exist? i was thinking something like

```typescript
// in index.ts
cron.schedule(crontab, () => {
    scraper.then(() => setupDb(true));
});

// in setup.ts
const setupDb = (update: boolean) => {
    // ... setup
    // if update and any of the tables are missing
   // then drop, create, populate the tables
   // else do nothing
}
```

i'll probably make the changes by tonight